.github/mtime_cache/action.js

```diff
@@ -0,0 +1,212 @@
+const { spawn } = require("child_process");
+const { dirname, resolve } = require("path");
+const { StringDecoder } = require("string_decoder");
+const { promisify } = require("util");
```

Comment:
Is there a reason to not use deno?

Reply:
Github actions can be written in node v12.x, or docker.

Reply:
Makes sense then. A comment at the top of the script would help future readers.